lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -35,9 +35,10 @@ nv_overlay_evpn:
 
 vn_segment_vlan_based:
   # MT-lite only
-  /N(3|9)/:
+  N3k: &vn_segment_vlan_based_N3k
     kind: boolean
     config_get: 'show running section feature'
     config_get_token: '/^feature vn-segment-vlan-based$/'
     config_set: 'feature vn-segment-vlan-based'
     default_value: false
+  N9k: *vn_segment_vlan_based_N3k
```

Comment:
Yeah, sorry, I had meant to do so. This is a YAML feature (http://www.yaml.org/spec/1.2/spec.html#id2785586) - basically think of it as aliases or pointers. `&label` provides an anchor to link against, while `*label` means 'this is a reference to that label'. So by doing it this way I'm basically saying 'the N9k values are identical to the N3k values' without needing to repeat myself and without needing to introduce special syntax for saying 'these are the values for N3k + N9k'. Clear as mud? :)